src/accountsdb/cache.zig

```diff
@@ -9,6 +9,7 @@ const LruCacheCustom = sig.common.lru.LruCacheCustom;
 const ReferenceCounter = sig.sync.reference_counter.ReferenceCounter;
 const Slot = sig.core.Slot;
 const Counter = sig.prometheus.counter.Counter;
+const ReadHandle = sig.accounts_db.buffer_pool.ReadHandle;
 
 /// Stores read-only in-memory copies of commonly used *rooted* accounts
 pub const AccountsCache = struct {
```

Comment:
do we even need the account_cache anymore? feels like all its logic is now in the bufferpool

Reply:
Probably not, it's likely superseded, will know for sure after benchmarks.

Reply:
nice, created a ticket here: https://github.com/orgs/Syndica/projects/2/views/10?pane=issue&itemId=94926926